src/helpers.py

```diff
@@ -65,7 +70,7 @@ def write_data_to_file(folder, filename, data, data_type=CONTENT_TYPE_TEXT):
                 sha256_hash_cur.update(byte_block)
 
         if sha256_hash_new.hexdigest() == sha256_hash_cur.hexdigest():
-            print(f"{timestamp()} Contents of {filename} haven't changed. Not overwriting existing file")
+            logger.warning(f"Contents of {filename} haven't changed. Not overwriting existing file")
```

Comment:
-> debug level, updates of config/map secrets not touching all entries can expected and should not create `WARNING` logs.

Reply:
agree